automated-comments/go/raindrops/entry_signature_changed.md

```diff
@@ -0,0 +1,4 @@
+The signature of the `Convert` function should be `Convert(i int) string`.
```

Comment:
```suggestion
Change the signature of the `Convert` function to be `Convert(i int) string`.
```

Perhaps?

Reply:
I applied it, but I will switch to the one in `general` soon anyway. There it says:

```
Fix the signature of the `%{function}` function. The expected signature is `%{signature}`.
```